bfapi/service/jobs.py

```diff
@@ -219,6 +208,26 @@ def create(
         tide_max_24h=tide_max,
     )
 
+def __genCmd(
```

Comment:
1. `__` prefix is not a convention used anywhere in the codebase and its use is considered somewhat controversial.
2. This particular usage of `__` prefix actually has no effect:

```
____ [~/code/_experiments/bfalg-ndwi-integration] ____ $ python3
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  5 2015, 21:12:44)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import bfapi.service.jobs as j
>>> j.__make_cli_cmd('pzsvc-ossim', ['alpha', 'bravo'])
'shoreline --image alpha,bravo --projection geo-scaled --threshold 0.5 --tolerance 0.075 shoreline.geojson'
>>>
```

As such, can you please change it to a single underscore?